spec/jobs/redeliver_web_hook_events_spec.rb

```diff
@@ -29,10 +38,37 @@
     ).to_return(status: 400, body: "", headers: {})
 
     messages =
-      MessageBus.track_publish { job.execute(web_hook: web_hook, web_hook_event: web_hook_event) }
+      MessageBus.track_publish { job.execute(web_hook: web_hook, web_hook_event: web_hook_event1) }
 
     expect(RedeliveringWebhookEvent.count).to eq(0)
     expect(messages.count).to eq(1)
     expect(messages.first.data).to include(type: "redelivered")
   end
+
+  context "when there is a redelivering_webhook_event in process" do
+    fab!(:redelivering_webhook_event_in_process) do
+      Fabricate(
+        :redelivering_webhook_event,
+        web_hook_event_id: web_hook_event2.id,
+        processing: true,
+      )
+    end
+
+    it "does not delete the webhook event in process" do
+      stub_request(:post, web_hook.payload_url).with(
+        body: "abc",
+        headers: {
+          "aa" => 1,
+          "bb" => 2,
+        },
+      ).to_return(status: 400, body: "", headers: {})
+
+      job.execute(web_hook: web_hook, web_hook_event: web_hook_event2)
```

Comment:
You don't pass anything into this job do you? It doesn't actually take any args right?